ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -418,7 +419,7 @@ struct FrameworkBuilder {
 
     // Verify Firebase headers include an explicit umbrella header for Firebase.h.
     let headersDir = podsDir.appendingPathComponents(["Headers", "Public", framework])
-    if framework.hasPrefix("Firebase") && framework != "FirebaseCoreDiagnostics" {
+    if framework.hasPrefix("Firebase"), framework != "FirebaseCoreDiagnostics" {
```

Comment:
This was from running `style.sh`

Reply:
That seems a lot less readable to me!  A comma operator instead of `&&`?!

Reply:
Yeah, I'll have to look into why that's the preferred one there. I think some `guard` statements make sense with `,` but this is less readable I'd say.